view/new.html

```diff
@@ -87,6 +87,19 @@ <h1 class="h2 crunch-top">Add a new URL</h1>
 	<div class="form-group clearfix">
 		<div class="row">
 			<div class="col-md-4 col-sm-4 col-xs-6">
+				<label class="control-label" for="new-task-actions">
+					Task Actions
+					(<a href="https://github.com/pa11y/pa11y#actions">see Pa11y documentation</a>)
+				</label>
+				<textarea class="form-control" id="new-task-actions" name="actions" placeholder="E.g. Click element #login-button">{{task.actions}}</textarea>
+				<em>(one action per line)</em>
```

Comment:
an aria-describedBy associated with that `<em>` added to the textarea would be naice